.travis.yml

```diff
@@ -9,10 +9,10 @@ addons:
     - libgtk-3-dev
     - libssh2-1-dev
 script:
-  - if [ "$TRAVIS_RUST_VERSION" == "stable" ]; then
-    rustup component add clippy;
-    cargo clippy --release;
-    fi
+  - test "$TRAVIS_RUST_VERSION" == "nightly" || rustup component add rustfmt clippy
+  - test "$TRAVIS_RUST_VERSION" == "nightly" || cargo clippy --all --release
+  - test "$TRAVIS_RUST_VERSION" == "nightly" || cargo check --all --release
+  - test "$TRAVIS_RUST_VERSION" == "nightly" || cargo fmt --all -- --check
```

Comment:
We use clippy on "stable" because is broke too many times on nightly.

Reply:
Yes, this runs on every version *but* `nightly`. I expressed it this way to ensure the list has a zero exit status when being skipped without suppressing an error from cargo.

Reply:
Oh, I missed this.
Why not combine clippy and check calls too?
Or there some cases where cargo check give new error/warning which clippy not give?

Reply:
Sorry, not checked changes before.
All good now.